data-prepper-plugin-framework/src/test/java/org/opensearch/dataprepper/plugins/test/TestComponent.java

```diff
@@ -0,0 +1,7 @@
+package org.opensearch.dataprepper.plugins.test;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class TestComponent {
```

Comment:
Let's add an integration test to verify that we can get some beans out of this context.

You can add this into the `DefaultPluginFactoryIT` test.

https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-core/src/integrationTest/java/org/opensearch/dataprepper/plugin/DefaultPluginFactoryIT.java

Reply:
Added an integration test now.